documentation/docs/98-reference/.generated/server-errors.md

```diff
@@ -14,10 +14,33 @@ You (or the framework you're using) called [`render(...)`](svelte-server#render)
 The `html` property of server render results has been deprecated. Use `body` instead.
 ```
 
-### lifecycle_function_unavailable
```

Comment:
let's revert this change and have a separate error for things-that-can't-be-called-in-the-browser. `hydratable.set` isn't a _lifecycle_ function, and 'on the browser' is very clunky